introduction.md

```diff
@@ -0,0 +1 @@
+Hi! I am myself and wont be saying more about that.

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.